lib/wallaby/query.ex

```diff
@@ -29,6 +29,7 @@ defmodule Wallaby.Query do
     * `:count` - The number of elements that should be found (default: 1).
     * `:visible` - Determines if the query should return only visible elements (default: true).
     * `:text` - Text that should be found inside the element (default: nil).
+    * `:get` - The number of the element to return if multimple elements satisfy the selection criteria.
```

Comment:
The problem with all these options is they could conceivably refer to a quantity rather than a specific instance.  `index` sounds a bit too 'techie' IMHO. (I still cling to the quaint notion that non-programmers could code tests.  How about `sequence`? Could be shortened to `seq` to help the lazy typists.